src/radio-group/__tests__/radio-group.test.tsx

```diff
@@ -110,6 +110,31 @@ describe('items', () => {
     expect(onChange).not.toHaveBeenCalled();
   });
 
+  test('can be readOnly', () => {
+    const { wrapper } = renderRadioGroup(<RadioGroup value="val1" name="test" items={defaultItems} readOnly={true} />);
+
+    expect(wrapper.getElement()).toHaveAttribute('aria-readonly', 'true');
+  });
+
+  test('does not trigger change handler if readOnly', () => {
+    const onChange = jest.fn();
+    const { wrapper } = renderRadioGroup(
+      <RadioGroup value="val1" name="test" items={defaultItems} readOnly={true} onChange={onChange} />
+    );
+
+    act(() => wrapper.findButtons()[0].findLabel().click());
+    expect(onChange).not.toHaveBeenCalled();
```

Comment:
I'm not sure this would have fired even if it wasn't readOnly. The test is trying to click the same item that is already selected so the value doesn't change.

Reply:
It actually fires when it's not `readOnly`, the component does not check if an element is selected, but I agrees that it might cause confusion, so changed `value` to `val2`